thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/completeness/checker/DataCompletenessUtil.java

```diff
@@ -0,0 +1,38 @@
+package com.linkedin.thirdeye.completeness.checker;
```

Comment:
We have a DataCompletenessTaskUtils.java. Maybe reuse that so we don't have too many Utils?